<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addMatchDialog</class>
 <widget class="QDialog" name="addMatchDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>210</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>210</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>210</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>addMatchDialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="finalLayout">
     <item>
      <layout class="QHBoxLayout" name="team1Layout">
       <item>
        <widget class="QLabel" name="team1Label">
         <property name="text">
          <string>Team 1:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="team1Input"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="team2Layout">
       <item>
        <widget class="QLabel" name="team2Label">
         <property name="text">
          <string>Team 2:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="team2Input"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="winnerLayout">
       <item>
        <widget class="QLabel" name="winnerLabel">
         <property name="text">
          <string>Winner:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="winnerInput"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="cancelButton">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="addButton">
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
